### PR TITLE
feat(mysql): sqlalchemy execute work

### LINKED ISSUE #4734: Work with sqlalchemy
**Summary**

```
mysql> SELECT @@tx_isolation;
+----------------+
| @@tx_isolation |
+----------------+
|              0 |
+----------------+
```

mysql-srv returns a int caused sqlalchemy panic:
```
> 486     return self.get_isolation_level(dbapi_conn)

File ~/.local/lib/python3.10/site-packages/sqlalchemy/dialects/mysql/base.py:2468, in MySQLDialect.get_isolation_level(self, connection)
   2466 if util.py3k and isinstance(val, bytes):
   2467     val = val.decode()
-> 2468 return val.upper().replace("-", " ")

AttributeError: 'int' object has no attribute 'upper'
```

The code line is:
https://github.com/datafuselabs/opensrv/blob/main/mysql/src/lib.rs#L458

```
                    if q.starts_with(b"SELECT @@") || q.starts_with(b"select @@") {
                        let w = QueryResultWriter::new(
                            &mut self.writer,
                            false,
                            self.client_capabilities,
                        );
                        let var = &q[b"SELECT @@".len()..];
                        let var_with_at = &q[b"SELECT ".len()..];
                        let cols = &[Column {
                            table: String::new(),
                            column: String::from_utf8_lossy(var_with_at).to_string(),
                            coltype: myc::constants::ColumnType::MYSQL_TYPE_LONG,
                            colflags: myc::constants::ColumnFlags::UNSIGNED_FLAG,
                        }];
```

Tasks:
- [x] Remove `SELECT @@` in `opensrv/blob/main/mysql/src/lib.rs`
- [x] Add a new check for `federated_server_setup_with_value`, key is sql and value is the constant value in a DataBlock
- [x] Add a new rule for `federated_server_setup_set_or_jdbc_command`
```
"(?i)^(ROLLBACK(.*))",
"(?i)^(SELECT @@(.*))",
```

### PR DESCRIPTION
**Pull request by BohuTANG**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

sqlalchemy execute work

## Changelog

- New Feature

## Related Issues

Fixes #4734

### CODE REVIEW

**Comment by BohuTANG** [comment]
@everpcpc need your help with the new python module install for stateless test

File: scripts/setup/dev_setup.sh [hunk 1]
```diff
@@ -446,7 +446,9 @@ if [[ "$INSTALL_DEV_TOOLS" == "true" ]]; then
 		install_pkg py3-pip "$PACKAGE_MANAGER"
 		install_pkg libffi-dev "$PACKAGE_MANAGER"
 	fi
-	python3 -m pip install --quiet boto3 "moto[all]" yapf shfmt-py mysql-connector-python toml
+	python3 -m pip install --quiet boto3 "moto[all]" yapf shfmt-py toml
+	# drivers
+	python3 -m pip install --quiet mysql-connector-python sqlalchemy
```
Comment (everpcpc) [inline scripts/setup/dev_setup.sh:451]: Moving this change to a new pull request and merge before would work for this.

**Comment by BohuTANG** [comment]
Will fix the failure test today

**Comment by BohuTANG** [comment]
The failure looks related to the python/or SQLAlchemy version, my local test is OK with:
```
python --version
Python 3.10.4

pip show SQLAlchemy
Name: SQLAlchemy
Version: 1.4.35
Summary: Database Abstraction Library
Home-page: https://www.sqlalchemy.org
Author: Mike Bayer
Author-email: mike_mp@zzzcomputing.com
License: MIT
Location: /home/bohu/.local/lib/python3.10/site-packages
Requires: greenlet
Required-by: ipython-sql
```

```
make stateless-test


09_0008_fuse_optimize_table:                                            [ OK ]
09_0009_remote_explain:                                                 [ OK ]
09_0010_remote_insert_overwrite:                                        [ OK ]
09_0011_create_table_with_opt_loc:                                      [ OK ]
09_0012_pushdown_limit:                                                 [ OK ]
10_0000_python_jdbc_driver:                                             [ OK ]
10_0001_python_sqlalchemy_driver:                                       [ OK ]
20_0000_describe_table:                                                 [ OK ]

```

**Pull request merged**